tests/ha/fencing.pm

```diff
@@ -25,8 +25,15 @@ sub run {
     check_cluster_state;
     barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
 
-    # Fence the master node
-    assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
+    # Fence the master node with sysrq or crm node fence
+    # Sysrq fencing is more a real crash simulation
+    if (get_var('USE_SYSRQ_FENCING')) {
+        record_info('Fencing info', 'Fencing done by sysrq');
```

Comment:
@alvarocarvajald  do you think this is useful? If so @juadk you may also add a `record_info` for the `crm node fence` method.

Reply:
For me it is enough with not seeing the `crm -F node fence` in node 2, but with the `record_info` it'll be easier to review, so +1 on adding a `record_info` in the else block.